resources/articles/adaptive_layer_height_threshold.md

```diff
@@ -0,0 +1,3 @@
+This setting can be used to influence the 'sensitiviy' of the adaptive layers algorithm.
+A lower value will result in a quicker reduction in layer height.
+Under water, this value is compared to the tangens of the steepest edge that crosses the current layer.
```

Comment:
The expression "under water" is in English only an expression for something having more financial debt than its worth. The Dutch idiom is better translated to "behind the scenes" or worded differently.